pylsp/plugins/symbols.py

```diff
@@ -3,6 +3,7 @@
 
 import logging
 import os
```

Comment:
This import is not necessary anymore, so please remove it to fix the problem reported by the `Static code analysis` workflow.